kofta/public/locales/el-GR/translation.json

```diff
@@ -49,7 +49,7 @@
       "featureText_2": "Ανοιχτές Εγγραφές",
       "featureText_3": "Cross-Platform Υποστήριξη",
       "featureText_4": "Ανοιχτού Πηγαίου Κώδικα",
-      "featureText_5": "Συνομιλία μέσω Τσατ",
+      "featureText_5": "Συνομιλία μέσω Chat",
```

Comment:
This should be changed back to greek since it doesn't make sense for it to be changed back.

Reply:
Because is looks better to to see Chat in English.